docs/pages/access-controls/login-rules/reference.mdx

```diff
@@ -21,9 +21,21 @@ Every login rule spec must contain either the `traits_map` field or a
 `traits_expression` field.
 
 They both serve the same purpose of transforming user traits.
-The difference lies only in the syntax you prefer for your use case, since you can write
+The logic difference lies only in the syntax you prefer for your use case, since you can write
 every `traits_map` as an equivalent `traits_expression`.
 
+`traits_map` will remove any traits not included while `trait_expression` will
+not. This is useful if you want to keep only a handful of traits while filtering
+everything else.
```

Comment:
I'm making this suggestion to avoid being too specific about the `traits_expression` behaviour since it can be used in many ways. If you look at the first example `traits_expression` below, traits not specifically mentioned **will** be filtered out.

```suggestion
`traits_map` will remove any traits not specifically included, while the
`traits_expression` syntax allows you add or modify only specific traits
while keeping the rest unchanged.
The `traits_map` behavior can be useful if you want to keep only a handful
of necessary traits while filtering out all others.
```